### PR TITLE
Multiple languages in Data API Docs

### PR DESCRIPTION
**Pull request by alexkunger**
adds multiple language support to Data API Docs

also removes sidebar for pages which don't use it

also adds Search to Data API Docs page

### CODE REVIEW

**Comment by alexkunger** [comment]
https://welkinhealth.atlassian.net/browse/DATA-1270

File: source/stylesheets/screen.css.scss [hunk 1]
```diff
@@ -379,7 +379,7 @@ html, body {
   }
 
   // the div is the tocify hidden div for placeholding stuff
```
Comment (josiah-welkin) [inline source/stylesheets/screen.css.scss]: comment still relevant?

File: source/data_api.html.md [hunk 2]
```diff
@@ -37,19 +55,82 @@ def get_token(client_id, client_secret, scope, endpoint, audience):
...
+try {
+  const res = await axios.get(url, {headers: headers});
+  const data = res.data.data;
+} catch (e) {
+  console.log("couldn't get data from Welkin");
+}
+```
+
+```shell
+CURL example not available
```
Comment (josiah-welkin) [inline source/data_api.html.md:133]: No curl example here?

**Review by josiah-welkin — APPROVED** [review]

File: source/data_api.html.md (hunk 2 above)
Comment (alexkunger) [reply source/data_api.html.md:133]: my CURL and bash skills are not good enough to make an JWT in bash :-(

**Pull request merged**